resources.html

```diff
@@ -5,16 +5,24 @@
 ---
 
 <h2 class="styleguide-header" id="instructions">Instructions and best practices</h2>
-<p>Quick start reference guide for using 18F brand assets</p>
-<img src="{{ site.baseurl }}/assets/img/Instructions_Preview.png">
+<p>Quick start reference guide for using 18F brand assets, including:</p>
+<ul>
+	<li>Logo usage</li>
+	<li>How to install the brand colors on your mac</li>
+	<li>Using our brand font</li>
+	<li>Keynote theme instructions</li>
+	<li>How to use the Google Slides template</li>
+<ul> <img src="{{ site.baseurl }}/assets/img/Instructions_Preview.png">
```

Comment:
Was this meant to be a `</ul>` instead of a `<ul>`?  Changing it fixes your layout bug!